packages/babel-types/src/retrievers/getOuterBindingIdentifiers.ts

```diff
@@ -0,0 +1,24 @@
+import getBindingIdentifiers from "./getBindingIdentifiers";
+import type * as t from "../types";
+
+export default function getOuterBindingIdentifiers(
+  node: t.Node,
+  duplicates: true,
+): Record<string, Array<t.Identifier>>;
+
+export default function getOuterBindingIdentifiers(
+  node: t.Node,
+  duplicates?: false,
+): Record<string, t.Identifier>;
+
+export default function getOuterBindingIdentifiers(
+  node: t.Node,
+  duplicates: boolean,
+): Record<string, t.Identifier | Array<t.Identifier>>;
```

Comment:
Same comment as for `packages/babel-types/src/retrievers/getBindingIdentifiers.ts`.